app/controllers/transport/border/active/RemoveBorderTransportYesNoController.scala

```diff
@@ -48,19 +50,29 @@ class RemoveBorderTransportYesNoController @Inject() (
     controllers.transport.border.active.routes.AddAnotherBorderMeansOfTransportYesNoController.onPageLoad(departureId, mode)
 
   def onPageLoad(departureId: String, mode: Mode, activeIndex: Index): Action[AnyContent] = actions
-    .requireIndex(departureId, BorderActiveSection(activeIndex), addAnother(departureId, mode)) {
+    .requireIndex(departureId, BorderActiveSection(activeIndex), addAnother(departureId, mode))
+    .andThen(getMandatoryPage.getFirst(IdentificationPage(activeIndex)))
+    .andThen(getMandatoryPage.getSecond(IdentificationNumberPage(activeIndex))) {
```

Comment:
`IdentificationNumberPage` might not always be populated

Scenario:

1. Do you want to add another active border transport means? Yes
2. Enter an identification type and submit
3. Click back
4. Click back
5. Click remove link -> redirects to tech difficulties because `IdentificationNumberPage` is unpopulated



One easy way around this is to define a case class with an associated reads method, similar to what we did in unloading
https://github.com/hmrc/manage-transit-movements-unloading-frontend/tree/main/app/models/removable

Reply:
@jackheslop96 done, ready for re-review